util/Migrator/DbScripts/2020-06-16_00_OrgSso.sql

```diff
@@ -0,0 +1,12 @@
+﻿IF OBJECT_ID('[dbo].[SsoConfig]') IS NULL
+BEGIN
+    CREATE TABLE [dbo].[SsoConfig] (
+        [OrganizationId]     UNIQUEIDENTIFIER    NULL,
+        [Identifier]         NVARCHAR (50)       NULL,
+        [Data]               NVARCHAR (MAX)      NULL,
+        [CreationDate]       DATETIME2 (7)       NOT NULL,
+        [RevisionDate]       DATETIME2 (7)       NOT NULL,
+        CONSTRAINT [FK_SsoConfig_Organization] FOREIGN KEY ([OrganizationId]) REFERENCES [dbo].[Organization] ([Id]) 
+    );
+END
+GO
```

Comment:
You need to add the create view (if not exists), and also need to add DROP if exists + CREATE scripts for your new stored procedures here as well.

Reply:
It looks like we're also performing a DROP if exists for views elsewhere - does that pattern not apply here?

Reply:
That would apply here, yes.